app/assets/javascripts/util/url.js.coffee

```diff
@@ -147,6 +139,18 @@ this.edsc.util.url = do(window
     new ChildCompressor('pg', new CwicGranuleIdListCompressor('cx'))
   ]
 
+#   new ParamFlattener(['science_keywords_h', '0', 'category'], 'fsc', false)
+  keywords = []
+  for index in [0...6]
```

Comment:
With the change of CMR-3207, the ```index``` can be any integer, picked by CMR, as long as they indicates hierarchical relationships between ancestor/child facets. It is highly unlikely but possible that variable_level_1 could look like: ```science_keywords_h[34][variable_level_1]```. I can't think of a way to address that issue though (maybe using regex?).